packages/theme/src/colors/alert.ts

```diff
@@ -76,6 +76,18 @@ export const informative = {
   informative50,
 }
 
+export type SuccessColor = keyof typeof success
+export type WarningColor = keyof typeof warning
+export type CriticalColor = keyof typeof critical
+export type InformativeColor = keyof typeof informative
+export type AlertColor = SuccessColor | WarningColor | CriticalColor | InformativeColor
+
+export const alertColors = {
+  ...success,
+  ...warning,
+  ...critical,
+  ...informative,
+}
```

Comment:
Is it possible to not have to export the colors spread but do it all in `colors/index.ts`?

Reply:
The code would be messy because we need to import them all.
I think this is the correct place to export.